wolfcrypt/src/pkcs12.c

```diff
@@ -670,6 +670,33 @@ int wc_d2i_PKCS12(const byte* der, word32 derSz, WC_PKCS12* pkcs12)
     return ret;
 }
 
+/* Convert WC_PKCS12 struct to allocated DER buffer.
+ * pkcs12 : non-null pkcs12 pointer
+ * der    : pointer-pointer to der buffer holding PKCS12
+ * return size of DER on success and negative on failure.
+ */
+int wc_i2d_PKCS12(WC_PKCS12* pkcs12, byte** der)
+{
+    int ret;
+    byte *buf = NULL;
+
+    if (pkcs12 == NULL) {
```

Comment:
Der should be ! NULL as well since you deference Der below.

Reply:
Check